### ISSUE TITLE
Release Twisted 13.0

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6353]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)| @therve reported|
|-|-|
|Trac ID|trac#6353|
|Type|task|
|Created|2013-03-06 09:37:11Z|
|Branch|https://github.com/twisted/twisted/tree/releases/release-13.0.0-6353|

It'd be nice to have a release before the dreaded pycon sprints.

<details><summary>Searchable metadata</summary>

```
trac-id__6353 6353
type__task task
reporter__therve therve
priority__normal normal
milestone__ 
branch__releases_release_13_0_0_6353 releases/release-13.0.0-6353
branch_author__ 
status__closed closed
resolution__fixed fixed
component__release_management release management
keywords__ 
time__1362562631000000 1362562631000000
changetime__1365005145000000 1365005145000000
version__None None
owner__therve therve
cc__radix
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)|@therve removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_4"></a>|@glyph set owner to @therve|
|-|-|

Unfortunately [ticket:5989 there was a regression].

Other than that this branch looks like it is a worthwhile release branch.

I think you should cut a new branch, and issue a pre2; however, since the only issue here is a regression in the test suite, not application code, you should be able to do an accelerated cycle rather than an entire additional week.

Please submit for re-review when a new release branch is ready.

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_5"></a>|@therve removed owner|
|-|-|

Alright, pre2 cut, revert included. The branch still conflicts about the quite file, but we have to deal with it manually I think.

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_6"></a>|@glyph commented|
|-|-|

[Just checking to see if that revert went OK.](http://buildbot.twistedmatrix.com/boxes-supported?branch=/branches/releases/release-13.0.0-6353)

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_7"></a>|@glyph commented|
|-|-|

[Oops that completely was the wrong URL.](http://buildbot.twistedmatrix.com/boxes-supported?branch=branches/releases/release-13.0.0-6353)

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_8"></a>|@glyph commented|
|-|-|

Uh, never mind, apparently I can't force builds, because the bzr mirror (and therefore the build farm) can't see release branches?  https://bugs.launchpad.net/twisted-trac-integration/+bug/1163438

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_9"></a>|@glyph set owner to @therve|
|-|-|

Now that the aforementioned bug is fixed, builds look OK.  I say: land it and ship it.

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_10"></a>|@therve set status to `closed`|
|-|-|

(In [37924]) Merge release-13.0.0-6353

Author: therve
Reviewer: glyph
Fixes: [#6353](https://github.com/twisted/twisted/issues/6353)

Release Twisted 13.0.0
